Branch-SDK/src/main/java/io/branch/referral/ServerRequestInitSession.java

```diff
@@ -11,6 +11,7 @@
 import org.json.JSONObject;
 
 import java.util.concurrent.locks.ReentrantLock;
+import java.util.logging.Logger;
```

Comment:
This looks like another unused import, but don't we ordinarily use `android.util.Log`?